widget/image.go

```diff
@@ -53,3 +60,143 @@ func (im Image) Layout(gtx layout.Context) layout.Dimensions {
 
 	return dims
 }
+
+type ImageSequences struct {
+	// Fit specifies how to scale the image to the constraints.
+	// By default it does not do any scaling.
+	Fit Fit
+	// Position specifies where to position the image within
+	// the constraints.
+	Position layout.Direction
+	// Frames is the sequence images, each image must be independent
+	// from the previous one and only one will be drawed per frame
+	Frames []paint.ImageOp
+	// FramesEnds control when each frame ends. The FramesEnds must
+	// have the lenght of Frames.
+	FramesEnds []time.Duration
+	// LoopCount controls the number of times an animation will be
+	// restarted during display.
+	// A LoopCount of 0 means to loop forever.
+	// A LoopCount of -1 means to show each frame only once.
+	// Otherwise, the animation is looped LoopCount+1 times.
```

Comment:
That `-1`, `0` and them `+1` is confusing. That comes from `image/gif`, to I keep the same.